ddtrace/internal/debug.py

```diff
@@ -23,6 +24,10 @@
 
 logger = get_logger(__name__)
 
+# The architecture function spawns the file subprocess on the interpreter
+# executable. We make sure we call this once and cache the result.
+architecture = callonce(lambda: platform.architecture())
```

Comment:
🤔  now I can't remember why I didn't do
```suggestion
architecture = callonce(platform.architecture)
```